api/texts.go

```diff
@@ -0,0 +1,57 @@
+package api
+
+import (
+	"github.com/gin-gonic/gin"
+	"github.com/joschahenningsen/TUM-Live/dao"
+	"github.com/joschahenningsen/TUM-Live/model"
+	"github.com/joschahenningsen/TUM-Live/tools"
+	"net/http"
+	"strconv"
+)
+
+func configTextRouter(router *gin.Engine, wrapper dao.DaoWrapper) {
+	routes := markdownTextRoutes{wrapper}
+	api := router.Group("/api")
+	{
+		api.Use(tools.Admin)
+		api.PUT("/texts/:id", routes.updateText)
+	}
+}
+
+type markdownTextRoutes struct {
+	dao.DaoWrapper
+}
+
+type updateTextDao struct {
+	Name       string `json:"name"`
```

Comment:
Does the name serve any purpose?

Reply:
The name is used on the admin tab to differentiate between texts, but other than that, no.